### ISSUE TITLE
topBar title not announced as heading/header in TalkBack

### ISSUE THREAD

**Issue by imgiseverything** [wix/react-native-navigation#7963]
### What happened?

In our app (Eurostar),  we are using v7.40.1 of react-native-navigation and the title in the topBar reads out its inner text content but doe snot inform the screen-reader that it is a header or heading as it does in iOS/VoiceOver

![Screenshot of the Eurostar app's login screen](https://github.com/user-attachments/assets/5a7bb1cc-bf80-4046-bf77-4b66201a6e1a)

### What was the expected behaviour?

With TalkBack turned on on Android tapping on the title inside the topBar should announce the title e.g. 'Log in, heading' or 'Log in, header'

### Was it tested on latest react-native-navigation?

- [x] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### Help us reproduce this issue!

Open the Eurostar app (latest version) and find a screen with a topBar - the log-in screen is a good example.

Link: https://play.google.com/store/apps/details?id=com.eurostar.androidapp&hl=en_GB&pli=1

### In what environment did this happen?

React Native Navigation version: 7.40.1
React Native version: 0.74.6
Has Fabric (React Native's new rendering system) enabled: (yes/no) 
Node version: 20.10.0
Device model: Galaxy A21s / SM-A2117F/DSN
Android version: S(Android 12)

**Comment by mensonones** [comment]
I believe that reproduction should be a project that reproduces the problem and not the application with the problem.
